lib/qiita/markdown/filters/user_input_sanitizer.rb

```diff
@@ -76,7 +76,7 @@ def node
             "h4"         => %w[id],
             "h5"         => %w[id],
             "h6"         => %w[id],
-            "img"        => %w[alt height src title width],
+            "img"        => %w[alt height src title width data-canonical-src],
```

Comment:
`HTML::Pipeline::CamoFilter` inserts `data-canonical-src` not a user. So no need to permit the attribute here